### DISCUSSION TITLE
Feature request: allow searcher to exclude arbitrary miners

### DISCUSSION THREAD

**Discussion by thegostep** [flashbots/pm#58] [searchers]
Some searchers have requested the ability to exclude certain miners from receiving bundles due to slight deviations in implementation leading to undesirable results.

While FlashBots is working hard to make sure all miners implementation is up to standard, it is possible for the miner to introduce breaking changes. The ability for the searcher to define a whitelist of miners would allow them to perform better risk management on execution.

This would take the form of an additional optional array parameter to `eth_sendBundle` which provides a list of accepted block.coinbase addresses.

Issue: https://github.com/flashbots/mev-relay-js/issues/38

**Comment by MicahZoltu** [comment]
Would one also be able to exclude certain miners who have shown bad behavior in the past, or only filter by MEV Geth version?

**Reply by thegostep** [reply]
Yes, this would provide the ability for searcher to do 'self policing' and exclude miners they do not trust.

API versioning will be included separately in the v0.2 release.
